src/ingestion/process-event.ts

```diff
@@ -475,17 +475,34 @@ export class EventsProcessor {
             )
         }
 
-        this.celery.sendTask('ee.tasks.webhooks_ee.post_event_to_webhook_ee', [
-            {
-                event,
-                properties,
-                distinct_id: distinctId,
-                timestamp,
-                elements_list: elements,
-            },
-            team.id,
-            siteUrl,
-        ])
+        let sendHookTask = !!team.slack_incoming_webhook
+        if (!sendHookTask) {
+            try {
+                const hookQueryResult = await this.db.postgresQuery(
+                    `SELECT COUNT(*) FROM ee_hooks WHERE team_id = $1 AND event = 'action_performed' LIMIT 1`,
+                    [team.id]
+                )
+                sendHookTask = !!hookQueryResult.rows[0].count
+            } catch (error) {
+                // In FOSS PostHog ee_hook does not exist. If the error is other than that, rethrow it
```

Comment:
Could it make sense to just check for `KAFKA_ENABLED` here and if so, disable the entire thing? These failing postgres queries might cause too much log noise for some users perhaps?

Also, on EE, this query might be run hundreds of times each second and perhaps could be cached somehow? That said, this is likely good enough for now and the caching can move into another PR if/when needed.

Reply:
Good point about `KAFKA_ENABLED`.
As for caching, I refactored this to use Redis for 120 s caching per team.